bankv2.rb

```diff
@@ -1,42 +1,35 @@
-# Update the Account class to be able to handle all of these fields from the CSV file used as input.
-# For example, manually choose the data from the first line of the CSV file
-# and ensure you can create a new instance of your Account using that data
-# Add the following class methods to your existing Account class:
-  # self.all - returns a collection of Account instances, representing all of the Accounts described in the CSV.
-  #See below for the CSV file specifications
-  # self.find(id) - returns an instance of Account where the value of the id field in the CSV matches
-#the passed parameter
-
 require 'csv'
 
 module Bank
 
   class Account
 
-    attr_reader :balance, :id, :owner, :open_date
+    attr_reader :id, :balance, :open_date, :owner
 
-#create instance variable for balance?
     def initialize (id, balance, open_date)
         if balance < 0
```

Comment:
Watch out for your indentation here - you don't need an additional indent when you start your `if` statement, only inside the block of it